Client/Application/AppDelegate.swift

```diff
@@ -10,98 +10,43 @@ import MessageUI
 import SDWebImage
 import SyncTelemetry
 import LocalAuthentication
-import SyncTelemetry
 import Sync
 import CoreSpotlight
 import UserNotifications
 import Account
-
-#if canImport(BackgroundTasks)
- import BackgroundTasks
-#endif
+import BackgroundTasks
```

Comment:
Great! I thought this was needed for one of our other targets. Glad we're cleaning up

Reply:
I could be mistaken but I believe since we now support only iOS 13 and up this is no longer needed. The iOS 13 checks for the code that uses this import were removed a while a go but this macro was likely just missed in that clean up.